.github/workflows/main.yml

```diff
@@ -37,4 +37,5 @@ jobs:
     - name: lint
       run: |
         yarn install --frozen-lockfile
+        yarn prettier-format
```

Comment:
does this actually create a commit with formatted changes? if not then I don't think it's necessary to have Github running prettier here